presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,184 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
 Old task is defined as a PrestoTask which has not received heartbeat for at least
 ``old-task-cleanup-ms``, or is not running and has an end time more than
 ``old-task-cleanup-ms`` ago.
+
+
+Session Properties
+------------------
+
+Defines all system session properties supported by native worker to ensure
+that they are the source of truth and to differentiate them from Java based
+session properties.
+
+The following are the native session properties kept as the source of truth in prestissimo.
+
+``driver_cpu_time_slice_limit_ms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``1000``
+
+Native Execution only. ``driver_cpu_time_slice_limit_ms`` defines the maximum CPU time,
+in milliseconds, that a driver thread is permitted to run before it must yield to other threads,
+facilitating fair CPU usage across multiple threads.
+A positive value enforces this limit, ensuring threads do not monopolize CPU resources.
+Setting this property to ``zero`` allows a thread to run indefinitely without yielding,
+which is not recommended in a shared environment as it can lead to resource contention.
+Negative values are considered invalid and are treated as a request to use the system default setting,
+which is ``1000`` ms in this case.
+
+``legacy_timestamp``
+^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Native Execution only. Use legacy TIME & TIMESTAMP semantics.
```

Comment:
```suggestion
Native Execution only. Use legacy TIME and TIMESTAMP semantics.
```
Avoid the ampersand (&) character. For context and suggestions see [the Gitlab documentation style guide recommended word list entry for &](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#).